### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by xunih**
## Related issue(s)
#72

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    - echo no | android create avd --force -n test -t $ANDROID_TARGET --abi armeabi-v7a # Create AVD for given api
+    - emulator -avd test -no-audio -no-window & # Start emulator
+    - android-wait-for-emulator
+    - adb devices # Display list of devices
+    - adb shell input keyevent 82 &
+    # Set `gradlew` as executable
+    - chmod +x $TRAVIS_BUILD_DIR/NewSpeechProject/gradlew
+    - cp $TRAVIS_BUILD_DIR/NewSpeechProject/gradle.properties.no.git $TRAVIS_BUILD_DIR/NewSpeechProject/gradle.properties
+    - sed -i "s/mykey/$API_KEY/g" $TRAVIS_BUILD_DIR/NewSpeechProject/gradle.properties
+    - sed -i "s/mysecret/$API_SECRET/g" $TRAVIS_BUILD_DIR/NewSpeechProject/gradle.properties
```
Comment (platisd) [inline .travis.yml]:
```yaml
    - cp $TRAVIS_BUILD_DIR/NewSpeechProject/gradle.properties.no.git $TRAVIS_BUILD_DIR/NewSpeechProject/gradle.properties
    - sed -i "s/mykey/$API_KEY/g" $TRAVIS_BUILD_DIR/NewSpeechProject/gradle.properties
    - sed -i "s/mysecret/$API_SECRET/g" $TRAVIS_BUILD_DIR/NewSpeechProject/gradle.properties
```

You don't need these three lines. The job will fail.

**Review by platisd — DISMISSED** [review]

**Comment by xunih** [comment]
Now the only failure is the problem of our code in arduino. I was wondering whether we should integrate the latest version of LineFollowing function and ObstacleAvoidance function and delete other useless files and try it again. It seems like now they are separate and then it will have the error of "Redefinition" @platisd

**Comment by platisd** [comment]
Both will work. Either place them in separate directories so arduino treats them as separate files or remove the unnecessary ones.

**Comment by xunih** [comment]
> Both will work. Either place them in separate directories so arduino treats them as separate files or remove the unnecessary ones.

Okay, will do it tomorrow, thank you!

**Comment by xunih** [comment]
Seems like we passed it now, but it still shows there is a failing check, how can we fix it? @platisd

**Comment by platisd** [comment]
This is an old branch. It is telling you that this branch (`push`) fails but if you'd merge (`pr`) it it would not. If you want all checks to pass you should either rebase this branch according to master or merge master into it.

The first can be done by:
```bash
git checkout xunih-patch-1
git rebase master
git push -f
```

The second one by:
```bash
git checkout xunih-patch-1
git merge master
git push
```

And there's probably a way to do both via GitHub but I don't know how.

**Review by flyslowly — APPROVED** [review]
CI settings have been fixed

**Pull request merged**